### DISCUSSION TITLE
Remove social icons

### DISCUSSION THREAD

**Discussion by itkkk** [tcbutler320/Jekyll-Theme-Dumbarton#52] [Q&A] [answered]
Hi guys,
how can i remove social icons for social i didn't use?

**Comment by tcbutler320** [comment, marked as answer]
Hey @itkkk, sorry I missed this! You're correct, the conditional formatting used in social.html was not properly working. I just fixed this issue and pulled #53 to the master branch and this issue should be fixed. Just leave the social empty in `_config.yml` and restart the development server, and you should see the social icon disappear from the nav bar. 

The issue was caused by inproper liquid formatting in social.html and social-footer.html

*Old Bad Code* 
> I used `else if` which is not proper liquid syntax. In liquid, it should be `elsif`
```html
   {% if site.twitter %}
        <a class="social-link" href="https://twitter.com/{{ site.twitter }}">
        <span style="color: {{ theme.social-color-footer }};">
            <i class="fab fa-twitter-square fa-lg"></i></a>
        </span>
    {% else if site.github %}
        <a href="https://github.com/{{ site.github }}">
        <span style="color: {{ theme.social-color-footer }};">
            <i class="fab fa-github-square fa-lg"></i></a>
        </span>
[SNIPPED]
```

*New Working Code* 
> I'm now avoiding using else entirely. Not sure why {% if %}, {% elseif %}, {% endif %} isnt working, but for now just using a series of if, endif statements works just fine.
```html
{% if site.twitter %}
        <a class="social-link" href="https://twitter.com/{{ site.twitter }}">
        <span style="color: {{ theme.social-color-footer }};">
            <i class="fab fa-twitter-square fa-lg"></i></a>
        </span>
    {% endif %}
  {% if site.github %}
      <a href="https://github.com/{{ site.github }}">
      <span style="color: {{ theme.social-color-footer }};">
          <i class="fab fa-github-square fa-lg"></i></a>
      </span>
  {% endif %}
```
